rust/Cargo.lock

```diff
@@ -260,7 +236,7 @@ dependencies = [
  "log",
  "num_cpus",
  "pairing",
- "rand_core 0.6.3",
+ "rand_core",
```

Comment:
This lost a version number, and so might fit in a lower or older version while still compiling.

Should we consider adding the former version back in, or finding the newest version and stating it explicitly?

Reply:
I don't see a change in `Cargo.toml` that directly explains this.   Is this change due to a change in `zingolib`?

https://github.com/zingolabs/zingo-mobile/issues/73